### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by Jolieresearch** [zrguo/MPLMM#4]
Can you open up the data you processed for the experiment?

**Comment by TianBaoGe** [comment]
Yes, I also prefer that the authors open up the dataset. Otherwise, it’s very difficult to reproduce the experimental results.

**Comment by zrguo** [comment]
The extracted features are provided under the corresponding repositories of the papers cited in the feature extraction part of the paper, and can be downloaded directly, e.g., MOSEI and MOSI can be obtained under the [mult](https://github.com/yaohungt/Multimodal-Transformer) repository. The rest can also be downloaded under the repository of the corresponding paper. In addition, you can use any feature extraction method, as long as you make sure that the dataset has the corresponding feature dimension and missing code functions.
